src/main/java/org/ow2/proactive/catalog/repository/entity/metamodel/CatalogObjectEntityMetaModelEnum.java

```diff
@@ -45,6 +45,7 @@
     KIND,
     LAST_COMMIT_TIME,
     NAME,
+    BUCKET_NAME,
```

Comment:
Just to be sure, we keep BUCKET_ID, NAME, and BUCKET_NAME? Do we keep the three of them?

Reply:
Done. BUCKET_ID is deleted now.